packages/react/src/Bento/Bento.tsx

```diff
@@ -0,0 +1,298 @@
+  const colorModeProp = colorMode ? {'data-color-mode': colorMode} : {}
+
+  const validChildren = React.Children.toArray(children).reduce<ValidItemChildren>((acc, child) => {
+    if (React.isValidElement(child) && typeof child.type !== 'string') {
+      if (child.type === Visual) {
+        acc.push(child as React.ReactElement<BentoVisualProps>)
+      }
+      if (child.type === Content) {
+        acc.push(child as React.ReactElement<BentoContentProps>)
+      }
+    }
+    return acc
+  }, [])
+
+  const orderedChildren = returnOrderedChildren({styleOrder, children: validChildren, currentBreakpointSize})
+
+  return (
+    <div
+      {...colorModeProp}
+      className={clsx(
+        ...bentoItemClassArray,
+        visualAsBackground && styles[`Bento__Item--visual-as-background`],
+        className,
+      )}
+      {...rest}
+    >
+      {orderedChildren}
+    </div>
+  )
+}
+
+type BentoContentProps = {
+  leadingVisual?: ReactElement | Icon
+  padding?: Padding | ResponsivePadding
+  verticalAlign?: Align | ResponsiveAlign
+  horizontalAlign?: ReducedAlign | ResponsiveReducedAlign
+  fixedBottomLink?: boolean
+} & React.HTMLAttributes<HTMLDivElement> &
+  BaseProps<HTMLDivElement>
+
+const Content = ({
+  children,
+  leadingVisual: LeadingVisual,
+  padding = 'normal',
+  verticalAlign = 'start',
+  horizontalAlign = 'start',
+  fixedBottomLink = false,
+  className,
+  ...rest
+}: BentoContentProps) => {
+  const bentoContentClassArray = [styles.Bento__Content]
+  bentoContentClassArray.push(
+    ...returnClassBasedOnResponsiveMap('Bento__Item', 'verticalAlign', verticalAlign),
+    ...returnClassBasedOnResponsiveMap('Bento__Item', 'horizontalAlign', horizontalAlign),
+    ...returnClassBasedOnResponsiveMap('Bento', 'padding', padding),
+  )
+  const HeadingChild = React.Children.toArray(children).find(
+    child => React.isValidElement(child) && child.type === _Heading,
+  )
+
+  const TextChild = React.Children.toArray(children).find(child => React.isValidElement(child) && child.type === Text)
+  const LinkChild = React.Children.toArray(children).find(child => React.isValidElement(child) && child.type === Link)
+  return (
+    <div className={clsx(styles[`Bento-padding--${padding}`], ...bentoContentClassArray, className)} {...rest}>
+      {React.isValidElement(LeadingVisual) &&
+        React.cloneElement(LeadingVisual as React.ReactElement<IconProps>, {
+          className: styles['Bento__Content-icon'],
+          size: LeadingVisual['size'] || 44,
+        })}
+      {React.isValidElement(HeadingChild) && React.cloneElement(HeadingChild as React.ReactElement<BentoHeadingProps>)}
+      {React.isValidElement(TextChild) &&
+        React.cloneElement(TextChild as React.ReactElement<TextProps>, {
+          variant: TextChild.props.variant || 'muted',
+          as: TextChild.props.as || 'p',
+          size: TextChild.props.size || '300',
+          className: clsx(styles['Bento_Content-text'], TextChild.props.className),
```

Comment:
For consistency, can we use the same naming convention please? Will need to be updated in a few different places.

```diff
- Bento_Content-text
+ Bento__Content-text
```

Reply:
This has been updated in the following [SHA](https://github.com/primer/brand/pull/390/commits/1f2a7cbc6aab5b74777ef577ee3bb68bc2b7cd3e#diff-9d379f39a3608e961347e076e7bd6000ff5c4ffbe3cbac8186b6565f8831c4f0)